UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Models/NoticesToMariners/ShowWeeklyFilesResponseModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace UKHO.MaritimeSafetyInformation.Common.Models.NoticesToMariners
+{
+    public class ShowWeeklyFilesResponseModel
+    {
+        public List<ShowFilesResponseModel> ShowFilesResponseModel { get; set; }
+        public List<YearWeekModel> YearAndWeek { get; set; }
```

Comment:
Would these be better named like?:
```suggestion
        public List<ShowFilesResponseModel> ShowFilesResponseList { get; set; }
        public List<YearWeekModel> YearAndWeekList { get; set; }
```

Reply:
Implemented suggested change